tests/gnrc_tcp/tests/05-garbage-pkts.py

```diff
@@ -0,0 +1,146 @@
+        dst_ll = get_riot_ll_addr(child)
+        dst_l2 = get_riot_l2_addr(child)
+        port = 33400 #generate_port_number()
+
+        # Setup RIOT Node wait for incoming connections from host system
+        child.sendline('gnrc_tcp_tcb_init')
+        child.expect_exact('gnrc_tcp_tcb_init: argc=1, argv[0] = gnrc_tcp_tcb_init')
+        child.sendline('gnrc_tcp_open_passive AF_INET6 {}'.format(port))
+        child.expect(r'gnrc_tcp_open_passive: argc=3, '
+                     r'argv\[0\] = gnrc_tcp_open_passive, '
+                     r'argv\[1\] = AF_INET6, argv\[2\] = (\d+)')
+        assert int(child.match.group(1)) == port
+
+        try:
+            print("- {} ".format(func.__name__), end="")
+            if child.logfile == sys.stdout:
+                func(child, tap, host_ll, dst_if, dst_l2, dst_ll, port)
+                print("")
+            else:
+                try:
+                    func(child, tap, host_ll, dst_if, dst_l2, dst_ll, port)
+                    print("SUCCESS")
+                except Exception as e:
+                    print("FAILED")
+                    raise e
+        finally:
+            child.sendline('gnrc_tcp_close')
+
+    return runner
+
+
+@testfunc
+def test_short_payload(child, src_if, src_ll, dst_if, dst_l2, dst_ll, dst_port):
+    # see https://github.com/RIOT-OS/RIOT/issues/11999
+    with socket.socket(socket.AF_INET6, socket.SOCK_STREAM) as sock:
+        sock.settimeout(child.timeout)
+        addr_info = socket.getaddrinfo(dst_ll + '%' + src_if, dst_port,
+                                       type=socket.SOCK_STREAM)
+        sock.connect(addr_info[0][-1])
+        child.expect_exact('gnrc_tcp_open_passive: returns 0')
+        child.sendline('gnrc_tcp_recv 1000000 1')
+        child.expect_exact('gnrc_tcp_recv: argc=3, '
+                           'argv[0] = gnrc_tcp_recv, '
+                           'argv[1] = 1000000, argv[2] = 1')
+        assert 1 == sock.send(b"f")
+        child.expect_exact('gnrc_tcp_recv: received 1', timeout=20)
+    verify_pktbuf_empty(child)
+
+
+@testfunc
+def test_short_header(child, src_if, src_ll, dst_if, dst_l2, dst_ll, dst_port):
+    # see https://github.com/RIOT-OS/RIOT/issues/12086
+    tcp_hdr = TCP(dport=dst_port, flags="S", sport=2342, seq=1, dataofs=6)
+    # shorten header
+    tcp_hdr = raw(tcp_hdr)[:-2]
+    sendp(Ether(dst=dst_l2) / IPv6(src=src_ll, dst=dst_ll) / tcp_hdr,
+          iface=src_if, verbose=0)
+
+    # let server command return to later check packet buffer
+    with socket.socket(socket.AF_INET6, socket.SOCK_STREAM) as sock:
+        sock.settimeout(child.timeout)
+        addr_info = socket.getaddrinfo(dst_ll + '%' + src_if, dst_port,
+                                       type=socket.SOCK_STREAM)
+        sock.connect(addr_info[0][-1])
+        child.expect_exact('gnrc_tcp_open_passive: returns 0')
+    verify_pktbuf_empty(child)
+
+
+@testfunc
+def test_send_ack_instead_of_syn(child, src_if, src_ll,
```

Comment:
Please rephrase, I don't know what you mean.... But regarding readability, I don't think this unreadable. The parameters are even divided up by category.

Reply:
Well I think is more readable if parameter lists are on one line if possible.

And since we have here a line length of max 119 chars, it might be possible that the entire signature fix on a single line.

Reply:
> Well I think is more readable if parameter lists are on one line if possible.

I guess we have to agree to disagree on that.

Reply:
> > Well I think is more readable if parameter lists are on one line if possible.
> 
> I guess we have to agree to disagree on that.

Yeah since this is more about aesthetics than anything else, I'll tolerate it :D